fuse/data/tokenizers/modular_tokenizer/op.py

```diff
@@ -30,6 +30,7 @@ def __init__(
         validate_ends_with_eos: Optional[bool] = True,
         eos: Optional[str] = "<EOS>",
         verbose: Optional[bool] = False,
+        on_unknown_default_value: Optional[str] = None,
```

Comment:
Why None? set it directly to "warn".
Lets remove the option to set it to None also in the other function and just support "warn" or "raise"